### PR TITLE
1602 split cdi fats

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_D_V4MAAnEei-3vT9-bzFwg was good. But there are merge conflicts. 

I am goiong to take out the commit that removes the fat_all bucket, as that has the commits, and use the above build to demonstrate that the new buckets are all good. Then I'll start a new branch to remove fat_all.

File: dev/com.ibm.ws.cdi_fat_annotations/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi_fat_annotations/bnd.bnd]: copyright year 2018

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
Build https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_d_HZUAQLEeiIX6lO4t6XVQ&tab=com.ibm.team.build.web.ui.internal.editors.result.test.TestProvider 

was good

**Pull request merged**